.github/workflows/buildwin.yml

```diff
@@ -15,13 +16,21 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.17.3
+      - name: Set up Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
       - name: Normal check out code
         uses: actions/checkout@v2
         with:
           submodules: true
       - name: Set GOBIN
         run: |
           echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
+      - name: Build wasmvm
+        working-directory: ./third_party/wasmvm/libwasmvm
```

Comment:
I think we can download a released source tarball of libwasmvm on the fly instead of maintaining a submodule.

Reply:
They don't have released version for windows. Anyway, I don't think we need to build `libwasm` to test windows build. We can just build `chainmain` with `sys_wasmvm` flag to test build. I'm trying it out. What do you think?

Reply:
For release, we'll have to package it together.

Reply:
that would be possible after https://github.com/CosmWasm/wasmvm/pull/288

Reply:
or do you mean to publish a tarball in our own fork in the meantime?

Reply:
I mean, sth like this:
```
curl -OL https://github.com/CosmWasm/wasmvm/archive/refs/tags/v1.0.0.tar.gz
tar xfz ...
cargo build ...
```

Reply:
We've forked `wasmvm`. No need for submodule anymore.